packages/tradershub/src/features/cfd/flows/MT5/AddedMT5AccountsList/AddedMT5AccountsList.tsx

```diff
@@ -21,10 +21,9 @@ const AddedMT5AccountsList = ({ account }: { account: THooks.MT5AccountsList })
     );
 
     const marketTypeDetails = MarketTypeDetails(isEU)[account.market_type ?? MarketType.ALL];
-
-    const title = marketTypeDetails?.title;
     const isVirtual = account.is_virtual;
-
+    const demoTitle = activeTradingAccount?.is_virtual ? 'Demo' : '';
+    const title = `${marketTypeDetails.title} ${demoTitle}`;
```

Comment:
This 2 lines are being hard coded in many places, you'll need to move them to a constants file.
Same with the word `Demo`.

Reply:
@matin-deriv I added a helper function for this can check it out and let me know if we are good here or any other suggestion, thanks 🙏